app/permissions_migrations.go

```diff
@@ -0,0 +1,154 @@
+package app
```

Comment:
This file is the key part, where all the permissions migrations are really done.